### DISCUSSION TITLE
[ 회의 일지 ] 2021-11-07

### DISCUSSION THREAD

**Discussion by Dong-Hyeon-Yu** [InhaBas/Inhabas.com#80] [회의기록]
#### [ @InhaBas/frontend ]
- @Yejin0O0 vue todo app 만들어봄.
- @leejaewoooong vue 학습
#### [ @InhaBas/backend ]
- @MingyeomKim 아키텍쳐 설계 중
- @Dong-Hyeon-Yu 권한 시스템 재설계

#
### 회의 내용
1.  **#4 로 인해 기존에 장고가 빌드하던 정적파일들.** 
    - **[현재] :** 빌드시 정적 파일에 임의의 해쉬값를 붙인다. => 추후 변경된 파일만 해쉬값을 변경하여 클라이언트가 정적파일을 잘 요청하도록 하고 있음.
     - **[방향] :** 현 시스템이 백엔드와 프론트가 restful 하게 완전히 분리되는 것이 목적
     - **[문제점] :** 현 방식대로면, 프론트에서 해쉬키까지 포함하여 경로요청을 해야한다. => 불가능 => 새로운 static files 관리 전략 필요 #60 참고
     - **[키워드]** WAS / WS / build / staticfile hashing table / staticfile versioning  

2. **장고 템플릿에 vue 컴포넌트를 붙일까 vs RESTful API 로 분리할까..?**
    - RESTful API 로 가자! 추후 어플 개발도 계획이 있는데, 장고 템플릿 엔진에 의존하면 어플 개발시 의존도가 너무 높아질 듯. 지금부터 분리해서 프론트의 자율성을 높이자

#
#### [ @InhaBas/frontend ]
- [x] todo app 갈무리
- [x] 게시판 vue 전환 작업 시작
- [ ] 정적 파일 관리 전략 고민
- [x] vue 아키텍쳐 정의
#### [ @InhaBas/backend ]
- [ ] django api 아키텍쳐 설계 마무리
- [x] 퍼미션 관련 데이터베이스 설정 적용 / sql 파일

**Comment by Yejin0O0** [comment]
정리왕 유동현
